modules/openthread/Kconfig.thread

```diff
@@ -175,6 +175,13 @@ config OPENTHREAD_MAC_STAY_AWAKE_BETWEEN_FRAGMENTS
 	help
 	  This optimization is done at the expense of power consumption on SED/SSED devices.
 
+config OPENTHREAD_DEFAULT_RX_SENSITIVITY
+	int "OpenThread default RX sensitivity in dBm"
+	range -100 -81 if NRF_802154_RADIO_DRIVER
```

Comment:
Why limiting the values here? I'd suggest removing this line.

Reply:
Values bigger or equal to 0 do not make much sense.
Noise floor gets set to `OPENTHREAD_DEFAULT_RX_SENSITIVITY` 
`int8_t SubMac::GetNoiseFloor(void) const { return Get<Radio>().GetReceiveSensitivity(); }`
And is then used to compute link margin as `int8_t linkMargin = aRss - aNoiseFloor`